### PR TITLE
[Bugfix] Add unreachable condition

### PR DESCRIPTION
**Pull request by ajanikow**

### CODE REVIEW

File: pkg/deployment/resources/pod_inspector.go [hunk 1]
```diff
@@ -218,6 +218,20 @@ func (r *Resources) InspectPods(ctx context.Context, cachedStatus inspectorInter
...
 		// End of Topology labels
 
+		if state, ok := r.context.MemberState(memberStatus.ID); ok {
+			if state.Reachable {
+				if memberStatus.Conditions.Update(api.ConditionTypeReachable, true, "ArangoDB is reachable", "") {
+					updateMemberStatusNeeded = true
+					nextInterval = nextInterval.ReduceTo(recheckSoonPodInspectorInterval)
+				}
+			} else {
+				if memberStatus.Conditions.Update(api.ConditionTypeReachable, true, "ArangoDB is not reachable", "") {
```
Comment (informalict) [inline pkg/deployment/resources/pod_inspector.go:228]: true -> false

File: pkg/deployment/member/state.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+		if _, err := c.Version(nctx); err != nil {
+			results[id].Reachable = false
+			return
+		}
+
+		results[id].Reachable = true
+	})
+
+	current := map[string]State{}
```
Comment (informalict) [inline pkg/deployment/member/state.go:75]: what is the idea of `current` if it is local variable and it is not passed anywhere?

**Review by informalict — COMMENTED** [review]
see comments

**Comment by ajanikow** [comment]
Fixes: https://github.com/arangodb/kube-arangodb/issues/887

**Pull request merged**